pkg/types/spec/validations.go

```diff
@@ -592,6 +612,9 @@ func validateContainers(
 			if strings.HasPrefix(key, "CORTEX_") || strings.HasPrefix(key, "KUBEXIT_") {
 				return errors.Wrap(ErrorCortexPrefixedEnvVarNotAllowed("CORTEX_", "KUBEXIT_"), strconv.FormatInt(int64(i), 10), userconfig.EnvKey, key)
 			}
+			if key == "HOST_IP" {
+				return errors.Wrap(ErrorDisallowedEnvVars(key), strconv.FormatInt(int64(i), 10), userconfig.EnvKey, key)
+			}
```

Comment:
Why is this necessary?

Reply:
We export this for the user containers. If we'd allow `HOST_IP` as an env var in the API spec, the user might accidentally overwrite this. This is just a precaution - unlikely that the user will overwrite, but still a possibility.

Reply:
I think we don't need to export `HOST_IP` to user containers anymore, just the proxy